<launch>
  <node pkg="rs_ros_sensor_board" type="rs_sensor_board_node" name="rs_sensor_board_node" output="screen">
     <!-- Serial port [Default "/dev/ttyUSB0"] -->
     <param name="serial_port" value="/dev/ttyUSB0"/>
     
     <!-- Serial baudrate [Default 115200] -->
     <param name="baudrate" value="115200"/>

     <!-- Serial timeout (msec) [Default 500] -->
     <param name="timeout_msec" value="500"/>

    <!-- Simulation mode [Default false] -->
    <param name="simul" value="false" />
   </node>   
</launch>
